benchmark/lib/Streamly/Benchmark/Prelude.hs

```diff
@@ -2141,6 +2151,8 @@ o_n_space_serial_toList value =
           -- , benchIOSink value "toPure" toPure
           -- , benchIOSink value "toPureRev" toPureRev
                 ]
+          , benchIOSink (value `div` 10) "classifySessionsOf"
```

Comment:
Change `classifySessionsOf` to `classifySessionsOf/10` maybe to indicate that you're using something other than the default value. It's not a big deal though.